std/array.d

```diff
@@ -3891,3 +3891,231 @@ unittest
     assert(appS.data == "hellow");
     assert(appA.data == "hellow");
 }
+
+version(all)
+{
+
+import std.traits : CommonType, Unqual, isStaticArray;
+import std.conv : emplaceRef;
+
+// @nogc: https://issues.dlang.org/show_bug.cgi?id=18439
+//nothrow @safe pure:
+nothrow:
+@safe:
+pure:
+
+/++
+Returns a static array constructed from `a`. The type of elements can be
+specified implicitly (`int[2] a = staticArray(1,2);`) or explicitly
+(`float[2] a = staticArray!float(1,2)`).
+D20180214T203015: The result is an rvalue, therefore uses like
```

Comment:
> D20180214T203015

What is this? A date? What for?

Reply:
done (was meant as a simple GUID to cross-reference; but removed it)